### PR TITLE
23H2KR5 - Intelligent Search API

### PR DESCRIPTION
**Pull request by julia-rabello**
#### Types of changes
- [x] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Intelligent Search API.json [hunk 1]
```diff
@@ -364,7 +1185,7 @@
             "hideUnavailableItems": {
                 "in": "query",
                 "name": "hideUnavailableItems",
-                "description": "Whether the result should hide unavailable items (`true`), or not (`false`)",
+                "description": "Defines whether the result should hide unavailable items (`true`), or not (`false`)",
```
Comment (ricardoaerobr) [inline VTEX - Intelligent Search API.json:1188]: Adicionar ponto final.

File: VTEX - Intelligent Search API.json [hunk 2]
```diff
@@ -1293,10 +2289,38 @@
...
+                                        "PRICERANGE"
+                                    ]
+                                },
+                                "name": {
+                                    "type": "string",
+                                    "description": "Human-readable format of the facet key."
+                                },
+                                "hidden": {
+                                    "type": "boolean",
+                                    "description": "Defines whether the client-side should hide the facet (`true`) or not (`false`."
```
Comment (ricardoaerobr) [inline VTEX - Intelligent Search API.json:2315]: (`false`)

File: VTEX - Intelligent Search API.json [hunk 3]
```diff
@@ -285,6 +411,544 @@
                             "application/json": {
                                 "schema": {
                                     "$ref": "#/components/schemas/ProductSearch"
+                                },
+                                "example": {
+                                    "products": [
+                                        {
+                                            "cacheId": "sp-2000003",
+                                            "productId": "2000003",
+                                            "description": "With this beautiful design piece you will never miss an appointment anymore. Yay.",
```
Comment (ricardoaerobr) [inline VTEX - Intelligent Search API.json:420]: Estranho esse description..rs

**Review by ricardoaerobr — CHANGES_REQUESTED** [review]
Revisar comentários

File: VTEX - Intelligent Search API.json (hunk 3 above)
Comment (julia-rabello) [reply VTEX - Intelligent Search API.json:420]: Isso é um example, mas bem observado hahah! Vou mexer!

**Review by ricardoaerobr — DISMISSED** [review]
LGTM!

**Review by ricardoaerobr — APPROVED** [review]
LGTM!

**Pull request merged**